lib/html/pipeline/sanitization_filter.rb

```diff
@@ -72,7 +72,10 @@ class SanitizationFilter < Filter
         },
         :protocols => {
           'a'   => {'href' => ANCHOR_SCHEMES},
-          'img' => {'src'  => ['http', 'https', :relative]}
+          'img' => {
+            'src'      => ['http', 'https', :relative],
+            'longdesc' => ['http', 'https', :relative]
```

Comment:
Done in https://github.com/mastahyeti/html-pipeline/commit/3425d140287970d3c3e0d4214f5a1885311d6778.